packages/brand/Kaizen/Brand/Brand.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { assetUrl } from "../../../hosted-assets"
+
+interface BrandProps {
```

Comment:
nit/ You'll probably want to export these props. There _could_ be situations where a consumer might want to know them.